### ISSUE TITLE
doc for setting logstash_version

### ISSUE THREAD

**Issue by hyao** [python-beaver/python-beaver#250]
Thanks for such a nice app!

To help beaver newbies like myself, it would be of immense help to add a line or two about setting logstash_version in beaver.conf in the usage documentation, just before the user type:

beaver

Then new users would not get "invalid logstash version" message. Or setting a default value of 1 in beaver itself?

Thanks,

**Comment by MrMarvin** [comment]
Yes please! At least give it a default value and only print a warning!

It is not acceptable to not be able to use beaver without typing into some ini file in some directory first. Even the docs say one could call `beaver` without any arguments which unfortunately is not true. Even if one would format output as say GELF you still need this configuration parameter...
